lib/beats_pipeline_generator.rb

```diff
@@ -42,20 +44,20 @@ def generate_beats_pipeline(mapping)
         statement = { 'from' => affected_field, 'type' => type }
         fields_to_convert << statement
 
-      # elsif ['uppercase', 'lowercase', 'to_array'].include?(row[:format_action])
-      #   processor = {
-      #     'script' => {
-      #       'lang' => 'javascript',
-      #       'source' => ... # herein lies all the fun :-)
-      #       ignore_failure: true,
-      #     }
-      #   }
-
+      elsif ['parse_timestamp'].include?(row[:format_action])
+        pipeline << {
+          'timestamp' => {
+            'field' => row[:source_field],
```

Comment:
Could you include the destination field support for Beats as well? I see below you support it for Elasticsearch.